gdk/commands/component/project_utils.py

```diff
@@ -108,6 +108,8 @@ def get_project_config_values():
     component_build_config = component_config["build"]
     bucket = component_config["publish"]["bucket"]
     region = component_config["publish"]["region"]
+    encrypt = component_config["publish"].get("encrypt")
```

Comment:
What are the different possible formats of `encrypt` in the `gdk-config.json`? Could you add that to the description of the PR?

Reply:
Updated supported encrypt values in PR description.